src/rust/dbv2/migrations/2023-12-21-132732_annotations/up.sql

```diff
@@ -0,0 +1,9 @@
+-- Your SQL goes here
+CREATE TABLE annotations (
```

Comment:
Would it make more sense to have this be under a schema `grafana.annotations`?

Or maybe just a different table name like `grafana_event_annotations` or even `grafana_annotations`?

Also, can we add some kind of spec on the table definition that skips if trying to re-insert the same events (for when a new annotation gets added)?